client/my-sites/checkout/gsuite-nudge/index.jsx

```diff
@@ -36,29 +39,50 @@ export class GSuiteNudge extends React.Component {
 		selectedSiteId: PropTypes.number.isRequired,
 	};
 
+	isMounted = false;
+
+	componentDidMount() {
+		this.isMounted = true;
+	}
+
+	componentWillUnmount() {
+		this.isMounted = false;
+	}
+
 	handleSkipClick = () => {
-		this.props.handleCheckoutCompleteRedirect();
+		const getThankYouPageUrlArguments = {
+			siteSlug: this.props.siteSlug,
+			receiptId: this.props.receiptId,
+			cart: this.props.cart,
+		};
+		const url = getThankYouPageUrl( getThankYouPageUrlArguments );
+		page.redirect( url );
```

Comment:
Is this a pretty safe assumption for the redirect? I was looking through the previous handler and it seemed pretty complicated.

Reply:
`getThankYouPageUrl` was created by meticulously going through the redirect portion of `handleCheckoutCompleteRedirect` and replicating it, so it's at least as accurate as checkout. (And after this PR, we'll be able to remove `handleCheckoutCompleteRedirect` entirely!)